README.md

```diff
@@ -309,7 +309,7 @@ A curated list of awesome Java frameworks, libraries and software.
 
 *Libraries and frameworks for writing distributed and fault-tolerant applications.*
 
-* [Akka](http://akka.io) - Toolkit and runtime for building concurrent, distributed, and fault tolerant event-driven applications.
+* [Akka](http://akka.io) - Toolkit and runtime for building concurrent, distributed, and fault-tolerant, event-driven applications.
```

Comment:
I think that the comma between "fault-tolerant" and "event-driven" is unnecessary, since AFAICT "event-driven applications" is itself a full noun (or whatever the correct grammatical term is). :)